src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerGetEventsTest.java

```diff
@@ -0,0 +1,111 @@
+import uk.gov.hmcts.darts.common.entity.JudgeEntity;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+import uk.gov.hmcts.darts.testutils.data.EventHandlerTestData;
+
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static uk.gov.hmcts.darts.testutils.data.CaseTestData.createCaseAt;
+import static uk.gov.hmcts.darts.testutils.data.CourthouseTestData.someMinimalCourthouse;
+import static uk.gov.hmcts.darts.testutils.data.CourtroomTestData.createCourtRoomWithNameAtCourthouse;
+import static uk.gov.hmcts.darts.testutils.data.EventTestData.createEventWith;
+import static uk.gov.hmcts.darts.testutils.data.HearingTestData.createHearingWithDefaults;
+import static uk.gov.hmcts.darts.testutils.data.JudgeTestData.createJudgeWithName;
+
+@AutoConfigureMockMvc
+class CaseControllerGetEventsTest  extends IntegrationBase {
+
+    public static final String EVENT_NAME = "New Description";
+    public static final String EVENT_TEXT1 = "event1";
+    @Autowired
+    private transient MockMvc mockMvc;
+
+    private static String endpointUrl = "/cases/{caseId}/hearings/{hearingId}/events";
+
+    private static final OffsetDateTime SOME_DATE_TIME = OffsetDateTime.parse("2023-01-01T12:00Z");
+    private static final String SOME_COURTHOUSE = "some-courthouse";
+    private static final String SOME_COURTROOM = "some-courtroom";
+
+    private CourtCaseEntity case1;
+    private HearingEntity hearing1;
+    private EventEntity event1;
+    private EventHandlerEntity activeHandler;
+
+    @BeforeEach
+    void setupData() {
+        CourthouseEntity swanseaCourthouse = someMinimalCourthouse();
+        swanseaCourthouse.setCourthouseName(SOME_COURTHOUSE);
+
+        CourtroomEntity courtroom1 = createCourtRoomWithNameAtCourthouse(swanseaCourthouse, SOME_COURTROOM);
+
+        case1 = createCaseAt(swanseaCourthouse);
+        case1.setCaseNumber("Case1");
+
+        JudgeEntity judge = createJudgeWithName("aJudge");
+        hearing1 = createHearingWithDefaults(case1, courtroom1, LocalDate.of(2023, 5, 20), judge);
+
+        dartsDatabase.saveAll(hearing1);
+
+        activeHandler = EventHandlerTestData.someMinimalEventHandler();
+
+        dartsDatabase.saveAll(activeHandler);
+
+        event1 = createEventWith("eventName", EVENT_TEXT1, hearing1, OffsetDateTime.now(), activeHandler);
+        EventEntity event2 = createEventWith("eventName", "event2", hearing1, OffsetDateTime.now(), activeHandler);
+        dartsDatabase.saveAll(event1, event2);
+    }
+
+    @AfterEach
+    void tearDown() {
+        dartsDatabase.addToTrash(activeHandler);
+    }
+
+    @Test
+    void casesGetEventsGetEndpoint() throws Exception {
+        MockHttpServletRequestBuilder requestBuilder = get(endpointUrl, case1.getId(), hearing1.getId());
+        mockMvc.perform(requestBuilder).andExpect(MockMvcResultMatchers.status().isOk());
+    }
```

Comment:
Suggest this be removed as the test case is already covered by `casesGetEvents()`